### PR TITLE
Spark Applications guide added

### PR DESCRIPTION
**Pull request by ArapovXD**

### CODE REVIEW

**Review by g1ease — APPROVED** [review]
Good job!

File: Applications of Spark.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Applications of Spark
```
Comment (arybin93) [inline Applications of Spark.md]: I would like to suggest find some examples of applications from Russian companies and proejcts.
What do you think?

**Review by arybin93 — COMMENTED** [review]
Thank your for your work. I've added comment for futher discussion
I would like to remind that we need to create issue before and also follow our naming convention for [branch](https://github.com/nnstu-appmath/.github-private/blob/main/profile/README.md).

File: Applications of Spark.md (hunk 1 above)
Comment (ArapovXD) [reply Applications of Spark.md]: Issues added, branchnames changed, info about Spark in Russia added. Check new pull requests, please

**Pull request closed without merging**